examples/mnist-ddp/gpu/CreatDocker.bash

```diff
@@ -0,0 +1,6 @@
+echo Please enter the system password first, and then Docker password
```

Comment:
Remove all bash files

Reply:
I will remove CreatDocker.bash
But install.bash and variable.bash are necessary to run the example. Can I keep it?

Reply:
@DeeperMind It is expected that users would have installed pytorch-operator for running examples.  Install.bash and variable bash are operator installation steps which are documented in the official website.  we don't require them

Reply:
Agree with @johnugeorge 
I think we do not need the script since we assume the users already install Kubeflow via ksonnet

Reply:
I see. Let me delete them later.